### DISCUSSION TITLE
"Clean style" vs "Original Clean" style differences

### DISCUSSION THREAD

**Discussion by Emma-Walker** [BEEmod/BEE2-items#4016] [General]
Shouldn't walls of "Clean" style look like this ->
![1436414856](https://user-images.githubusercontent.com/8507237/142810758-470a0d77-34a2-4b82-9e66-e3395d68af84.jpg)

instead of like this ->
![20211122071623_1](https://user-images.githubusercontent.com/8507237/142810824-c2094660-398e-468c-821c-77d9f6e8df4b.jpg)

Is this a bug or was the style changed sometime ago? With the latest BEEMod versions I don't see a significant difference between "Original Clean" and "Clean"

**Comment by TeamSpen210** [comment]
Something broke it seems with the algorithm.

**Comment by vrad-exe** [comment]
Same bug as #4004 probably
